src/Arcus.Scripting.DevOps/Arcus.Scripting.DevOps.psm1

```diff
@@ -0,0 +1,21 @@
+<#
+ .Synopsis
+  Set a variable in the Azure DevOps pipeline at runtime.
+ 
+ .Description
+  Assign a value to a DevOps pipeline variable during the execution of this pipeline.
+
+ .Parameter VariableName
+  The name of the variable to set in the pipeline.
+ 
+ .Parameter VariableValue
+  The value of the variable to set in the pipeline.
+#>
+function Set-DevOpsVariable {
+	param(
+		[Parameter(Mandatory=$true)][string]$VariableName = $(throw "VariableName is required"),
+		[Parameter(Mandatory=$true)][string]$VariableValue = $(throw "VariableValue is required")
```

Comment:
I'd call this like this since Variable is in the command:
```suggestion
		[Parameter(Mandatory=$true)][string]$Value = $(throw "VariableValue is required")
```

Reply:
Agreed and let's update the parameter-name in the exception-message as well.

```suggestion
		[Parameter(Mandatory=$true)][string]$Value = $(throw "Value is required")
```